core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -35,6 +35,9 @@ private[spark] object ToolTips {
 
   val OUTPUT = "Bytes and records written to Hadoop."
 
+  val STORAGE_MEMORY =
+    "Total memory used for caching of the total available memory for caching.This is the storage memory used on the executor."
```

Comment:
I suggest something like "Memory used / total available for storage of data like RDD partitions cached in memory"?